ui/src/components/Issuer/Issuer.tsx

```diff
@@ -36,38 +34,46 @@ const Issuer: React.FC<Props> = ({ onLogout }) => {
     const ledger = useLedger();
 
     const registeredIssuer = useStreamQuery(RegisteredIssuer);
-    const issuerModel = useStreamQuery(IssuerModel);
-
     const allCustodianRelationships = useStreamQuery(CustodianRelationship).contracts.map(makeContractInfo);
 
     const [ profile, setProfile ] = useState<Profile>({
         'name': createField('', 'Name', 'Your full legal name', 'text'),
+        'location': createField('', 'Location', 'Your current location', 'text'),
```

Comment:
i feel like this should be a drop down or something - location as in zip code? State? City? not pressing but definitely vague